### PR TITLE
Add wikitude-cordova-plugin types

### PR DESCRIPTION
**Pull request by zbarbuto**
Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] Add or edit tests to reflect the change. (Run with `npm test`.)
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] Run `npm run lint package-name` (or `tsc` if no `tslint.json` is present).

Select one of these and delete the others:

If adding a new definition:
- [x] The package does not already provide its own types, or cannot have its `.d.ts` files generated via `--declaration`
- [x] If this is for an NPM package, match the name. If not, do not conflict with the name of an NPM package.
- [ ] Create it with `dts-gen --dt`, not by basing it on an existing project. (tried serveral times to use CLI but was getting errors so did base on an existing project
- [x] `tslint.json` should be present, and `tsconfig.json` should have `noImplicitAny`, `noImplicitThis`, `strictNullChecks`, and `strictFunctionTypes` set to `true`.

Related link (source for the types):

https://github.com/pbreuss/wikitude-ionic-3-starter-app/blob/master/src/app/WikitudePlugin.d.ts

### CODE REVIEW

File: types/com.wikitude.phonegap.wikitudeplugin/tslint.json [hunk 1]
```diff
@@ -0,0 +1,79 @@
+{
+    "extends": "dtslint/dt.json",
```
Comment (plantain-00) [inline types/com.wikitude.phonegap.wikitudeplugin/tslint.json]: For new packages, this file should only `"extends": "dtslint/dt.json"`

**Review by plantain-00 — CHANGES_REQUESTED** [review]

File: types/com.wikitude.phonegap.wikitudeplugin/tslint.json (hunk 1 above)
Comment (zbarbuto) [reply types/com.wikitude.phonegap.wikitudeplugin/tslint.json]: Whoops, fixed 👍

**Review by plantain-00 — APPROVED** [review]

**Comment by zbarbuto** [comment]
The readme says that master is automatically published to npm but I'm not getting any results for `@types/com.wikitude.phonegap.wikitudeplugin` - is it possible that it's having issues with the dots in the name?

**Comment by sandersn** [comment]
Could be. @andy-ms or @RyanCavanaugh is that something types-publisher might have trouble with?

**Comment by andy-ms** [comment]
The package has was published yesterday. `npm info @types/com.wikitude.phonegap.wikitudeplugin` succeeds.

**Comment by zbarbuto** [comment]
Yep working now, thanks 👍

**Pull request merged**